test/unit/org/apache/cassandra/service/QueryInfoTrackerTest.java

```diff
@@ -73,7 +73,7 @@ public void setupTest()
     public void testSimpleQueryTracing()
     {
         int keys = 4;
-        int clustering = 4;
+        int clustering = 7;
```

Comment:
this test has an assert for `keys + keys * clustering`.
keys == clustering is thus unfortunate. I changed it and the test still passes ;)